Скачихин Александр

0encodedguy0 opened the pull request:
Выполнены 1 и 2 задания

0encodedguy0 committed:
Add files via upload

0encodedguy0 committed:
Update 1_task.txt

0encodedguy0 committed:
Add files via upload

0encodedguy0 committed:
Update 1_task.txt

0encodedguy0 committed:
Update 1_task.txt

0encodedguy0 committed:
Update 1_task.txt

sevryukov commented on 1_task.txt:
```diff
@@ -0,0 +1,9 @@
+3. Функция F_EMPLOYEE_GET не обрабатывает случаи, когда пользователь не найден.
+4. Функция F_WORKS_LIST ссылается на функций F_WORKITEMS_COUNT_BY_ID_WORK и F_EMPLOYEE_FULLNAME.
+   Т.к. вызов каждой функции надо выполнить для каждой строки результирующей таблицы, это может привести к проблемам с производительностью.
+5. В таблицах используются поля с типом VARCHAR большого размера, что влияет на скорость обработки запросов.
```
Допустим. Какие предложите альтернативы?

sevryukov commented on 2_task.sql:
Не буду повторять комментарии к аналогичному скрипту в #4

sevryukov reviewed (commented):
Всё написал в #4

0encodedguy0 committed:
Add files via upload

0encodedguy0 committed:
Create PossibleDisadvantages

0encodedguy0 committed:
Update InsertingRandomValues.sql

За основу была взята работа Девришева Надира. Но я постарался чуть оптимизировать его код. Также, вместо функций я использую процедуры. О возможных недостаткая написано в файле PossibleDisadvantages. В файле ProceduresForInserting.sql содержится код процедур. В файле InsertingRandomValues.sql код заполнения таблиц случайными значениями

0encodedguy0 commented:
За основу была взята работа Девришева Надира. Но я постарался чуть оптимизировать его код. Также, вместо функций я использую процедуры. О возможных недостаткая написано в файле PossibleDisadvantages. В файле ProceduresForInserting.sql содержится код процедур. В файле InsertingRandomValues.sql код заполнения таблиц случайными значениями
